beluga_benchmark/scripts/profiling/profile_amcl_with_bagfile

```diff
@@ -34,8 +25,6 @@ if [[ "$KPTR_RESTRICT" -ne "0" || "$PERF_EVENT_PARANOID" -ne "-1" ]]; then
     echo 0 | sudo tee -a /proc/sys/kernel/kptr_restrict 1>/dev/null
 fi
 
-0<&- script -qefc "ros2 launch beluga_example common_nodes_launch.py" /dev/null &
-perf record -e cycles -F 99 -g --call-graph dwarf --\
-    /ws/install/beluga_amcl/lib/beluga_amcl/amcl_node --ros-args --log-level info --ros-args -r __node:=amcl \
-    -p use_sim_time:=True --params-file /ws/install/beluga_example/share/beluga_example/config/params.yaml &
-ros2 bag play /ws/install/beluga_example/share/beluga_example/bags/perfect_odometry --rate 3
+PERF_PREFIX="perf record -e cycles -F 99 -g --call-graph dwarf -o perf.data --"
+
+ros2 launch beluga_example perfect_odometry_launch.py localization_prefix:="${PERF_PREFIX}" playback_rate:=3
```

Comment:
```suggestion
ros2 launch beluga_example perfect_odometry_launch.py localization_prefix:="${PERF_PREFIX}" playback_rate:=1
```

Reply:
Fixed in https://github.com/Ekumen-OS/beluga/pull/211/commits/9bf5e843a51ed9871a59bd0e8ed8c38d4c437c2f.